src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* BUGFIX: Add new PE `CV_CFL_LANG` language code for `ALIASOBJ` and `Rust`. [530](https://github.com/microsoft/binskim/pull/530)
```

Comment:
>BUGFIX [](http://example.com/codeflow?start=2&length=6)

If this is a bugfix, what are we fixing and why? what was the behavior?

Reply:
originally I set as FEATURE,
but we think FEATURE is buying us much in this document.
I don't think BREAKING is good here to,

Reply:
@michaelcfanning and @VCOLAWTON is there a better label we can use for this type of code change for documentation purposes? FEATURE and BUGFIX both don't fit this scenario.